app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1605,8 +1611,9 @@ table li .account_details {
 // start non-admin items menu
 .items table {
   background-color: #202020;
-  margin-top: 50px;
-  border: 1px solid #303030;
+  margin-top: 100px;
```

Comment:
Properties should be ordered background-color, border, margin-bottom, margin-top